migrations/src/main/scala/za/co/absa/enceladus/migrations/migrations/MigrationToV1.scala

```diff
@@ -39,6 +39,8 @@ object MigrationToV1 extends MigrationBase with CollectionMigration with JsonMig
   createIndex("dataset", Seq(IndexField("name", ASC), IndexField("version", ASC)), unique = true)
   createIndex("schema", Seq(IndexField("name", ASC), IndexField("version", ASC)), unique = true)
   createIndex("mapping_table", Seq(IndexField("name", ASC), IndexField("version", ASC)), unique = true)
+
+  createIndex("run", Seq(IndexField("dataset", ASC), IndexField("datasetVersion", ASC)))
```

Comment:
I am not sure how MongoDB work, but isn't this index a subset/sub-index of the one right bellow? (with just little smaller selectivity). 🤔

Reply:
Yup, according to this [article](https://engineering.mixmax.com/blog/improving-mongo-performance-by-managing-indexes), the new one is redundant and MongoDB engine should be able to use the other index whenever the new indexed might seem useful.

Reply:
Aaaah, I did not realize it, but it actually makes sense 😊 .